doc/source/serialization.rst

```diff
@@ -64,6 +64,48 @@ Serialization notes
 
 - Lock objects are mostly unserializable, because copying a lock is meaningless and could cause serious concurrency problems. You may have to come up with a workaround if your object contains a lock.
 
+Troubleshooting
+---------------
+
+Use ``ray.util.inspect_serializability`` to identify tricky pickling issues. This function can be used to trace a potential non-serializable object within any Python object -- whether it be a function, class, or object instance.
```

Comment:
Can we mention `RAY_PICKLE_VERBOSE_DEBUG ` here as well?

Reply:
RAY_PICKLE_VERBOSE_DEBUG is not actually used in anywhere except Tune right now. I think we can do this once we actually use it in Ray core.